docs/root/configuration/http_conn_man/stats.rst

```diff
@@ -45,6 +45,7 @@ statistics:
    downstream_rq_tx_reset, Counter, Total request resets sent
    downstream_rq_non_relative_path, Counter, Total requests with a non-relative HTTP path
    downstream_rq_too_large, Counter, Total requests resulting in a 413 due to buffering an overly large body
+   downstream_rq_completed, Counter, Total responses
```

Comment:
What does completed mean? Can you be more specific about how it differs from _rq_total

Reply:
total is incremented on request initiation, regardless of response code (if any), whereas this is incremented on request completion
@btc and @danielhochman have the context here in terms of why they wanted it.
i can be more specific in the docs, ya, willfix